src/utilities/_word-break.scss

```diff
@@ -0,0 +1,8 @@
+
+.u-break--word {
+    word-break: break-word;
+}
+
+.u-break--all {
+    word-break: break-all;
+}
```

Comment:
Ik geloof dat dit ook op 1 regel kan. Net als in het float voorbeeld dat je gebruikte. Zodra er sprake is van meer properties, dan ga je ze onder elkaar zetten. Net als hier eigenlijk: https://github.com/zicht/zss/blob/master/docs/layout/float.md

Reply:
Ok, vroeg me al af welke van de 2 stylen voorkeur heeft, aangezien ik ze vaak beiden tegenkom :smile: